contrib/drivers/openstack/cinder/cinder.go

```diff
@@ -76,6 +79,36 @@ type CinderConfig struct {
 	Pool        map[string]PoolProperties `yaml:"pool,flow"`
 }
 
+//ListPoolOpts
+type ListPoolOpts struct {
+	// ID of the tenant to look up storage pools for.
+	TenantID string `q:"tenant_id"`
+	// Whether to list extended details.
+	Detail bool `q:"detail"`
+	//Volume_Type of the StoragePool
+	Volume_Type string `q:"volume_type"`
+}
+
+//Struct of Pools listed by volumeType
```

Comment:
space after //

Reply:
I get all volume types from the customer's configuration file:"/etc/opensds/driver/cinder.yaml"，Then I add them to a array  and Traversing this array, At last,I get pools by each volume type and add pools to another array and return all pools.